yt_dlp/extractor/triller.py

```diff
@@ -22,30 +28,28 @@ def _perform_login(self, username, password):
         if self._API_HEADERS.get('Authorization'):
             return
 
-        user_check = self._download_json(
+        self._API_HEADERS['Content-Type'] = 'application/json'
```

Comment:
I don't like how this mutates and then resets the var. I'd rather do:
```suggestion
        headers = {**self._API_HEADERS, 'Content-Type': 'application/json'}
```